python/README.md

```diff
@@ -12,71 +10,23 @@ $ pip install foxglove-websocket
-   **or:** Install the foxglove-websocket package and run `python -m foxglove_websocket.examples.simple_server`.
+```
+python -m foxglove_websocket.examples.simple_server
+```
 
-2. In a browser, open up https://studio.foxglove.dev and initiate a Foxglove WebSocket connection to `ws://localhost:8765/`.
+To see data from this server, open [Foxglove Studio](https://studio.foxglove.dev?ds=foxglove-websocket&ds.url=ws://localhost:8765/) with a Foxglove WebSocket connection to `ws://localhost:8765/`:
 
 <img width="676" alt="Foxglove Studio displaying data from the example server" src="https://user-images.githubusercontent.com/14237/145260376-ddda98c5-7ed0-4239-9ce4-10778ee8240b.png">
 
-```py
-import asyncio
-import json
-import time
-from foxglove_websocket import run_cancellable
-from foxglove_websocket.server import FoxgloveServer, FoxgloveServerListener
-from foxglove_websocket.types import ChannelId
-
-
-async def main():
-    class Listener(FoxgloveServerListener):
-        def on_subscribe(self, server: FoxgloveServer, channel_id: ChannelId):
-            print("First client subscribed to", channel_id)
+## Server template
 
-        def on_unsubscribe(self, server: FoxgloveServer, channel_id: ChannelId):
-            print("Last client unsubscribed from", channel_id)
-
-    async with FoxgloveServer("0.0.0.0", 8765, "example server") as server:
-        server.set_listener(Listener())
-        chan_id = await server.add_channel(
-            {
-                "topic": "example_msg",
-                "encoding": "json",
-                "schemaName": "ExampleMsg",
-                "schema": json.dumps(
-                    {
-                        "type": "object",
-                        "properties": {
-                            "msg": {"type": "string"},
-                            "count": {"type": "number"},
-                        },
-                    }
-                ),
-            }
-        )
-
-        i = 0
-        while True:
-            i += 1
-            await asyncio.sleep(0.2)
-            await server.send_message(
-                chan_id,
-                time.time_ns(),
-                json.dumps({"msg": "Hello!", "count": i}).encode("utf8"),
-            )
-
-
-if __name__ == "__main__":
-    run_cancellable(main())
-```
+Copy [this template code](https://github.com/foxglove/ws-protocol/blob/main/python/src/foxglove_websocket/examples/simple_server.py) into a file and run it (e.g. `python3 server.py`). Then, make the necessary adjustments to the file to customize this simple server to your desired specifications.
 
 ## Development
 
-When developing or maintaining the foxglove-websocket package, it is recommended to use [pipenv](https://github.com/pypa/pipenv) to manage development dependencies and virtualenv.
+Use [pipenv](https://github.com/pypa/pipenv) to manage developer dependencies and `virtualenv` while developing.
```

Comment:
I might prefer just reverting this line; I think it's a little confusing to just say "while developing" since it's not clear whether we mean while developing in (a fork of) this repo, or developing your own server. Pipenv is something I found to be relatively easy to use for the dev setup here, but others might not want to use it
```suggestion
When developing or maintaining the foxglove-websocket package, it is recommended to use [pipenv](https://github.com/pypa/pipenv) to manage development dependencies and virtualenv.
```